src/nlotrajectories/scripts/run_benchmark.py

```diff
@@ -45,14 +36,19 @@ def run_benchmark(config_path: Path):
         x_goal=x_goal,
         N=config.solver.N,
         dt=config.solver.dt,
-        sdf_func=obstacles.sdf,
+        sdf_func=obstacles.approximated_sdf,
         control_bounds=tuple(config.body.control_bounds),
         use_slack=config.solver.use_slack,
         slack_penalty=config.solver.slack_penalty,
     )
 
     X_opt, U_opt, _ = runner.run()
     plot_trajectory(X_opt, geometry, obstacles, title=config_path.stem, goal=config.body.goal_state)
+    plot_levels(obstacles.sdf, title=str(config_path.stem) + "_sdf")
+    if config.solver.mode == "l4casadi":
+        plot_levels(obstacles.approximated_sdf, title=str(config_path.stem) + "_nn_sdf")
+    else:
+        plot_levels(obstacles.approximated_sdf, title=str(config_path.stem) + "_math_sdf")
```

Comment:
not necessary, but maybe we could replace `if` with this:

```suggestion
    suffix = "_nn_sdf" if config.solver.mode == "l4casadi" else "_math_sdf"
    plot_levels(obstacles.approximated_sdf, title=f"{config_path.stem}{suffix}")
```